test/services/CashTransactionsServiceSpec.scala

```diff
@@ -105,7 +166,7 @@ class CashTransactionsServiceSpec extends SpecBase {
       )
       running(app) {
         val result = await(service.retrieveCashTransactionsSummary("can", dateFrom, dateTo))
-        val expectedResult = CashTransactions(Nil, Nil)
+        val expectedResult = CashTransactions(Nil, Nil, None)
```

Comment:
Please remove None as the third parameter as it is already defaulted to None. Same as line 257